### ISSUE TITLE
Tray icons very small compared to app icons

### ISSUE THREAD

**Issue by vimgouser** [home-sweet-gnome/dash-to-panel#1907]
I'm not sure if this is a bug or just that dash-to-panel is missing a feature.

This is what my toolbar looks like.

![dash to panel](https://github.com/home-sweet-gnome/dash-to-panel/assets/111732626/405ed2c4-efde-40be-b3ec-c54781c57d93)

Either there is a bug and the tray icons are not matching the size of app icons, or there is no way to alter the size of the tray icons (in which case, the tray icons are too small).

Any ideas?

**Comment by mihaaairu** [comment]
Have the same problem.

**Comment by charlesg99** [comment]
Please bring this to the creator of the tray extension

**Comment by HelgeBoehme** [comment]
I noticed that Dash to Panel behaves differently if the tray font size is set to non-default. Then the tray icon size is [set to the font size](https://github.com/home-sweet-gnome/dash-to-panel/blob/master/src/panelStyle.js#L129). This is much smaller than most theme's default (1.09em). I tried to patch 1.09em at this place but then the tray icon size is not scaled with the font size.  I'd suggest to add a multiplicator for the icon size (like 1.25, which works for me) or to have independent settings for tray font size and tray icon size.
Background: I'm using Gnome 48 on Wayland with no fractional scaling support on hidpi display, so I set the global font scaler to 1.5 (but the tray font needs to be a little smaller).
